src/Stratis.Bitcoin.Features.Api/ApiSettings.cs

```diff
@@ -9,9 +9,21 @@ namespace Stratis.Bitcoin.Features.Api
     /// </summary>
     public class ApiSettings
     {
+        /// <summary>The default port used by the API when the node runs on the bitcoin network.</summary>
+        public const int DefaultBitcoinApiPort = 37220;
```

Comment:
I think this should go in the Network class (next to rpc port)

Reply:
This is only for the API feature though. It does not really have an obvious place in the Network class.